### ISSUE TITLE
unhandledRejection: TypeError: Cannot read property 'push' of undefined

### ISSUE THREAD

**Issue by ex1st** [cdimascio/express-openapi-validator#415]
**Describe the bug**
Regression appears after #414

```javascript
// request.schema.preprocessor.js

preprocessPathLevelParameters(pathItemKey, pathItem) {
        var _a, _b, _c;
        const parameters = (_a = pathItem.parameters) !== null && _a !== void 0 ? _a : [];
        if (parameters.length === 0)
            return;
        const v = pathItem[pathItemKey];
        if (v === parameters)
            return;
        const ref = (_b = v === null || v === void 0 ? void 0 : v.parmeters) === null || _b === void 0 ? void 0 : _b.$ref;
        const operationParameters = (ref ? (_c = this.ajv.getSchema(ref)) === null || _c === void 0 ? void 0 : _c.schema : v.parameters);
        for (const param of parameters) {
            operationParameters.push(param); // <-- this place
        }
    }
```

**Actual behavior** 
Error throws if `parameters` property exists in `path` but doesn't exists in `pathItem`

**Expected behavior**
`pathItem` can be without `parameters` property

**Examples and context**
```json
{
  "openapi": "3.0.0",
  "paths": {
    "/product/{productId}/prices": {
      "parameters": [{
        "in": "path",
        "name": "productId",
        "required": true,
        "schema": {
          "$ref": "#/components/schemas/DualID"
        }
      }],
      "patch": {
        "summary": "Update price values.",
        "tags": ["Price"],
        "requestBody": {
          "required": true,
          "content": {
            "application/json": {
              "schema": {
              }
            }
          }
        }
      }
    }
  },
  "schemas": {
    "UniqID": {
      "type": "string",
      "pattern": "^([a-f0-9]){24}$",
      "description": "Hexadecimal unique identifier",
      "example": "5dee464910450609ca7ea2d7"
    },

    "InternalID": {
      "type": "string",
      "pattern": "^([A-Z0-9]){5,6}$",
      "description": "Short identifier that uniq for certain entity",
      "example": "XQ02F"
    },

    "DualID": {
      "oneOf": [{
          "$ref": "#/components/schemas/UniqID"
        },
        {
          "$ref": "#/components/schemas/InternalID"
        }
      ],
      "description": "Dual unique identifier, accepts long and short identifiers transparently"
    }
  }
}
```

**Comment by cdimascio** [comment]
fix in `v4.3.3`. thanks @krizzje for the pr

**Comment by cdimascio** [comment]
fixed and in `v4.3.4`. thanks @ex1st for the fix!
